test/integration/admin_user_creates_item_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class AdminUserCreatesItemTest < ActionDispatch::IntegrationTest
+  test "admin creates a item" do
+    admin = User.create(first_name: "Admin",
+                     last_name: "Admin",
+                     username: "admin",
+                     password: "password",
+                     role: 2)
+    user = User.create(first_name: "taylor",
+                      last_name: "moore",
+                      username: "taylor",
+                      password: "moore",
+                      role: 1)
+
+    category1, category2, category3 =
```

Comment:
Useless assignment to variable - `category2`. Use `_` or `_category2` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `category3`. Use `_` or `_category3` as a variable name to indicate that it won't be used.